data/ch/funfacts.yml

```diff
@@ -0,0 +1,25 @@
+################################# funfacts ##################################
+funfacts:
+  enable : true
+  bg_image : "images/backgrounds/17-042_main_image.jpg"
+  bg_image_webp : "images/backgrounds/17-042_main_image.webp"
+  counter_item:
+    # counter item loop
+    - title : "Happy Clients"
+      icon : "ti-face-smile" # themify icon pack : https://themify.me/themify-icons
+      count : "320"
+    
+    # counter item loop
+    - title : "Projects completed"
+      icon : "ti-archive" # themify icon pack : https://themify.me/themify-icons
+      count : "350"
+    
+    # counter item loop
+    - title : "Positive feedback"
+      icon : "ti-thumb-up" # themify icon pack : https://themify.me/themify-icons
+      count : "158"
+    
+    # counter item loop
+    - title : "Cups of Coffee"
+      icon : "ti-gift" # themify icon pack : https://themify.me/themify-icons
+      count : "250"
```

Comment:
We can remove this file (we'll add Chinese-language in a future PR using the go-18n for Hugo approach)